docs/source/user-guide/installation.md

```diff
@@ -30,22 +30,32 @@ Make sure the following requirements are met and software installed on your mach
 
 ### Supported Spark Versions
 
-Comet currently supports the following versions of Apache Spark:
+Comet currently supports the following versions of Apache Spark. Note that this documentation is for the main
+branch. Please refer to the version of this documentation in the appropriate GitHub release branch for released 
+Comet versions.
 
-- 3.4.x (Java 8/11/17, Scala 2.12/2.13)
-- 3.5.x (Java 8/11/17, Scala 2.12/2.13)
+We recommend only using Comet with Spark versions where we currently have both Comet and Spark tests enabled in CI.
+Other versions may work well enough for development and evaluation purposes.
+
+| Spark Version | Java Version | Scala Version | Comet Tests in CI | Spark Tests in CI |
+| ------------- | ------------ | ------------- | ----------------- | ----------------- |
+| 3.4.3         | 8/11/17      | 2.12/2.13     | Yes               | Yes               |
+| 3.5.4         | 8/11/17      | 2.12/2.13     | No                | Yes               |
+| 3.5.5         | 8/11/17      | 2.12/2.13     | Yes               | Yes               |
 
 Experimental support is provided for the following versions of Apache Spark and is intended for development/testing
 use only and should not be used in production yet.
 
-- 4.0.0-preview1 (Java 17/21, Scala 2.13)
+| Spark Version  | Java Version | Scala Version | Comet Tests in CI | Spark Tests in CI |
+| -------------- | ------------ | ------------- | ----------------- | ----------------- |
+| 4.0.0-preview1 | 11/17        | 2.13          | Yes               | Yes               |
```

Comment:
I think we only test with Java 17 for Spark 4.0 right now

Reply:
Thanks. Updated.